roles/sqs_queue/tasks/create.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Create fifo dead letter queue resources
+  command: aws sqs create-queue --queue-name "{{ sqs_dead_letter_queue.name }}.fifo" --attributes file://queue_attributes.json
```

Comment:
Is this idempotent?

Reply:
Running a create command with an existing queue name and attributes returns a url but does not affect the existing queue.

Running the create command with an existing queue name and an attribute difference returns an error that a queue exists with different attributes.

Running the create command with an existing queue name and an additional attribute returns an error that a queue exists with different attributes.